server/gitrest/packages/gitrest-base/src/utils/definitions.ts

```diff
@@ -10,6 +10,7 @@ export enum Constants {
 	StorageRoutingIdHeader = "Storage-Routing-Id",
 	StorageNameHeader = "Storage-Name",
 	IsEphemeralContainer = "Is-Ephemeral-Container",
+	SimplifiedCustomDataHeader = "SimplifiedCustomData",
```

Comment:
nit: `"Simplified-Custom-Data"` to follow the existing header name patterns here